core/arch/arm/plat-imx/platform_config.h

```diff
@@ -101,20 +101,16 @@
 #define CFG_TA_RAM_SIZE		ROUNDDOWN((TZDRAM_SIZE - CFG_TEE_RAM_VA_SIZE), \
 						CORE_MMU_DEVICE_SIZE)
 
-#define DEVICE0_PA_BASE		ROUNDDOWN(UART0_BASE, \
-						CORE_MMU_DEVICE_SIZE)
-#define DEVICE0_VA_BASE		(64 * 1024 * 1024)
-#define DEVICE0_SIZE		CORE_MMU_DEVICE_SIZE
-#define DEVICE0_TYPE		MEM_AREA_IO_NSEC
-
-/*
- * console uart virtual address
- * The physical address is 0x2020000, we mapped it to 0x4020000,
- * see DEVICE0_VA_BASE and DEVICE0_PA_BASE
- */
-#define CONSOLE_UART_BASE		(0x4020000)
+#define CONSOLE_UART_BASE		(UART0_BASE)
 #define CONSOLE_UART_PA_BASE		(UART0_BASE)
```

Comment:
Why do we have both `CONSOLE_UART_BASE` and   `CONSOLE_UART_PA_BASE`, I'd prefer to remove the latter.